Sources/HttpPipeline/ResponseHeader.swift

```diff
@@ -11,7 +11,8 @@ public enum ResponseHeader {
   case other(String, String)
   case wwwAuthenticate(Authenticate)
 
-  public enum CookieOption: Hashable, CustomStringConvertible {
+  public enum CookieOption: Hashable, CustomStringConvertible, Comparable {
```

Comment:
need comparable on this so that I can sort cookies into a well-defined order (since the cookie header uses a `Set` of these)